serviceaccounts.rb

```diff
@@ -2,6 +2,7 @@
 require "set"
 require_relative "workbench"
 require_relative "utils/common"
+require 'tmpdir'
```

Comment:
```suggestion
require "tmpdir"
```

nitpicking my own code: double-quotes for consistency with the others